lib/datasets.go

```diff
@@ -292,11 +248,11 @@ func (m *DatasetMethods) Get(p *GetParams, res *GetResult) error {
 			return err
 		}
 
-		if dr.Path == "" && res.FSIPath != "" {
+		if strings.HasPrefix(ref.Path, "/fsi") {
```

Comment:
Could this function, that checks if a ref.Path is an FSI path, and the one below, which trims the prefix, be put into a utility function? It's also being used in `loadDataset`, and I have a feeling it's going to show up a few other places as well.

Reply:
yes, the right place for this should be `qfs.PathKind`: https://github.com/qri-io/qfs/blob/a6a7b98f0fa0969f6eae50fe14c2c6b5b32f4b4f/fs.go#L76-L87

Need to PR in a fix there first tho. I'll add a `TODO`